### ISSUE TITLE
Creator: Hovering an arrow highlights it

### ISSUE THREAD

**Issue by brettfiedler** [phetsims/paper-land#164]
A suggestion from a codesigner: 

Hovering an arrow makes it highlight. We could probably have it increase in line width by a visually distinct amount.

We should also consider adding a tooltip on hover that indicates the components joined

e.g., "imgPositionController -> img1Position", "img1Position -> imgComponent"

**Comment by brettfiedler** [comment]
JG noted we could also do this for a hovered component to highlight all arrows connected to it. Let's investigate the above suggestion first, then default to the suggestion in this comment.

**Comment by brettfiedler** [comment]
For expediency, let's go with the simpler implementation and just highlight all arrows connected to a component when it is hovered.

**Comment by jessegreenberg** [comment]
OK, I went with the following.

1) When a component is being edited, connections in or out of that component have a thicker line width. Other connections have a thinner line width.
2) When there is no edit, all lines have the same default line width.
3) I used the active edit instead of 'hover' for ease and for better performance. However, if hover is better, we can do that instead easily now.
4) This doubles the number of canvas strokes so we may have a performance hit.
5) Line widths are in ViewConstants.js and can be adjusted if you want to play around with values.

![image](https://github.com/user-attachments/assets/e1b58e8d-1c85-4020-80af-23802e4ac275)
